src/main/java/org/scalameter/annotations/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.annotations;
```

Comment:
> If there are clashing members, I would alternatively just kill the problematic member in the old japi.

Does it mean that I should remove the old japi? I can do that, but I thought that we would made it as deprecated only.